src/components/DashboardPage/DashboardTask.tsx

```diff
@@ -0,0 +1,106 @@
+import styled from '@emotion/styled';
+
+import BtnTask from '@/components/common/BtnTask/BtnTask';
+import ScrollGradient from '@/components/common/ScrollGradient';
+
+interface DashboardTaskProps {
+	text: 'upcoming' | 'postponed' | 'inprogress';
+}
+
+function DashboardTask({ text }: DashboardTaskProps) {
+	const DASHBOARD_TASK_TYPE = {
+		UPCOMING: '다가오는 할 일',
+		POSTPONED: '지연된 할 일',
+		INPROGRESS: '진행중인 할 일',
+	};
+	let taskStatus = '';
+	if (text === 'upcoming') {
+		taskStatus = DASHBOARD_TASK_TYPE.UPCOMING;
+	} else if (text === 'postponed') {
+		taskStatus = DASHBOARD_TASK_TYPE.POSTPONED;
+	} else if (text === 'inprogress') {
+		taskStatus = DASHBOARD_TASK_TYPE.INPROGRESS;
+	}
+	return (
+		<TaskLayout>
+			<TextBox>
+				<TaskText text={text}>{taskStatus}</TaskText>
+				<NumberTextBox>
+					<Number>2</Number>
+					<NumberText>개</NumberText>
+				</NumberTextBox>
+			</TextBox>
+			<ScrollArea>
+				<BtnTask btnType={2} status="Done" isDescription={false} />
+				<BtnTask btnType={2} status="Done" isDescription={false} />
+				<BtnTask btnType={2} status="Done" isDescription={false} />
+				<BtnTask btnType={2} status="Done" isDescription={false} />
+				<BtnTask btnType={2} status="Done" isDescription={false} />
+				<BtnTask btnType={2} status="Done" isDescription={false} />
+				<BtnTask btnType={2} status="Done" isDescription={false} />
```

Comment:
```suggestion
				<BtnTask btnType='target' status="Done" isDescription={false} />
				<BtnTask btnType='target' status="Done" isDescription={false} />
				<BtnTask btnType='target' status="Done" isDescription={false} />
				<BtnTask btnType='target' status="Done" isDescription={false} />
				<BtnTask btnType='target' status="Done" isDescription={false} />
				<BtnTask btnType='target' status="Done" isDescription={false} />
				<BtnTask btnType='target' status="Done" isDescription={false} />
```


해당 부분의 btnType 값들이 `1, 2, 3`에서 `'staging' | 'target' | 'delayed'` 으로 수정되어서, 다음과 같이 변경 부탁드립니다!!